haskell/test/Test/Ligo/BaseDAO/Proposal/Tokens.hs

```diff
@@ -0,0 +1,102 @@
+-- SPDX-FileCopyrightText: 2021 TQ Tezos
+-- SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+-- | Contains tests on @propose@ entrypoin logic for testing the Ligo contract.
```

Comment:
This is a copy of the `Propose` sibling module, perhaps it could contain a more detailed description?

P.s. it has the same `entrypoin` typo